paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlActionUtils.java

```diff
@@ -79,13 +79,13 @@ static Connection getConnection(Configuration mySqlConfig, boolean tinyint1NotBo
             throws Exception {
         String url =
                 String.format(
-                        "jdbc:mysql://%s:%d%s",
+                        "jdbc:mysql://%s:%d?useInformationSchema=true%s",
```

Comment:
Should we use an option to open this feature?
I don't know how much impact it has in performance.

Reply:
the param in default `true` in  mysql 8.0.3 ([doc](https://dev.mysql.com/doc/connector-j/8.1/en/connector-j-connp-props-metadata.html)) ,but it is default `false` in mysql    5.7,  if we add an option and when we set it to `false` ,do    we should set it to `false` explicitly in mysql8 ?

Reply:
Default auto?

Reply:
Or default we can just disable table comments.